contrib/lkt/extensions/src/liblktlang-implementation-extensions.adb

```diff
@@ -127,11 +127,9 @@ package body Liblktlang.Implementation.Extensions is
      (Node : Bare_LK_Node;
```

Comment:
Commit message: negligible or neglectable, not neglectible (not a word)

Also commit message:
> This refactoring allows to hoist to Langkit_Support several type definitions/subprograms currently in templating code (analysis contexts,units and the "basic" node structure ...

First the paren is never close, second this suggests that you're doing it in this commit (which isn't the case), rt. "will allow". So I searched for those types and didn't find them :cry:

Reply:
Yes, indeed I’ll replace with “will allow”, sorry.